src/main/java/life/qbic/model/petab/IdWithPattern.java

```diff
@@ -0,0 +1,22 @@
+package life.qbic.model.petab;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class IdWithPattern {
```

Comment:
If the id has a pattern does it make sense to check for the pattern during generation?

Reply:
once we know more about it, yes